src/integration-test/kotlin/integration/WithdrawalIntegrationTest.kt

```diff
@@ -202,4 +211,81 @@ class WithdrawalIntegrationTest {
 
         Assertions.assertEquals(initialBalance + BigInteger.valueOf(amount.toLong()), token.balanceOf(toAddress).send())
     }
+
+    /**
+     * Try to withdraw to address in whitelist
+     * @given A notary client and withdrawal address. Withdrawal address is in whitelist
+     * @when client initiates withdrawal
+     * @then notary approves the withdrawal
+     */
+    @Test
+    fun testWithdrawInWhitelist() {
+        val integrationHelper = IntegrationHelperUtil()
+
+        val peer = "http://localhost:8080"
+
+        async {
+            notary.main(emptyArray())
+        }
+        Thread.sleep(5_000)
+
+        // create client
+        val clientId = integrationHelper.registerClient()
+        val clientKeypair = integrationHelper.irohaKeyPair
+
+        IntegrationHelperUtil().setWhitelist(clientId, arrayOf(toAddress, "qwerqt"))
```

Comment:
I don't think we need this new instance of IntegrationHelperUtil just to call setWhitelist. You may use integrationHelper  variable instead.